spec/requests/as_content_owner/content_owner_spec.rb

```diff
@@ -53,19 +53,15 @@
     describe '.includes(:claim)' do
       let(:videos) { $content_owner.videos.includes(:claim) }
       let(:video_with_claim) { videos.find{|v| v.claim.present?} }
+      let(:video_with_claim_with_asset) do
+        videos.find{|v| v.try(:claim).try(:asset)}
+      end
```

Comment:
I think we can get rid of this, and just use the previous `video_with_claim`. 

In other words, every video with claim will also have an asset, and we just expect it to be eager-loaded too.